From 2968afc0969ce2f809065b69fe0ce14885eed1f3 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Tue, 16 Apr 2019 08:49:07 +0900
Subject: [PATCH 1/4] Unifies the functionality provided by CMake to lowercase

This unify to lowercase, because CMake's functions are mixed lowercase and
uppercase.

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 CMakeLists.txt     | 2 +-
 src/CMakeLists.txt | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 118fc32..2b61b91 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,7 +5,7 @@ project (libubootenv C)
 # set (libubootenv_VERSION_MAJOR 1)
 # set (libubootenv_VERSION_MAJOR_VERSION_MINOR 0)
 set(VERSION	"0.1")
-SET(SOVERSION "${VERSION}")
+set(SOVERSION "${VERSION}")
 add_definitions(-DVERSION="${VERSION}")
 
 #set(CMAKE_C_FLAGS_DEBUG "-g")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 051732b..ea5979c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,19 +1,19 @@
 cmake_minimum_required (VERSION 2.6)
 # Sources and private headers
-SET(libubootenv_SOURCES
+set(libubootenv_SOURCES
   uboot_env.c
   uboot_private.h
 )
 
 # Public headers
-SET(include_HEADERS
+set(include_HEADERS
   libuboot.h
 )
 
 add_library(ubootenv SHARED ${libubootenv_SOURCES} ${include_HEADERS})
-SET_TARGET_PROPERTIES(ubootenv PROPERTIES SOVERSION ${SOVERSION})
+set_target_properties(ubootenv PROPERTIES SOVERSION ${SOVERSION})
 
-ADD_LIBRARY(ubootenv_static STATIC ${libubootenv_SOURCES} ${include_HEADERS})
+add_library(ubootenv_static STATIC ${libubootenv_SOURCES} ${include_HEADERS})
 add_executable(fw_printenv fw_printenv.c)
 add_executable(fw_setenv fw_setenv.c)
 target_link_libraries(fw_printenv ubootenv z)
-- 
2.20.1

